frontend/src/views/FuelSupplies/_schema.jsx

```diff
@@ -147,7 +148,7 @@ export const fuelSupplyColDefs = (optionsData, errors, warnings) => [
   {
     field: 'fuelCategory',
     headerComponent: RequiredHeader,
-    headerName: i18n.t('fuelSupply:fuelSupplyColLabels.fuelCategory'),
+    headerName: i18n.t('fuelSupply:fuelSupplyColLabels.fuelCategoryId'),
```

Comment:
Why are you changing the label to have a different name than the column?
fuelCategory

Reply:
How we are showing errors in the AG Grids? We are saving the records using the `useSaveFuelSupply` hook which sends a POST request to backend. Then Pydantic validates the payload, if there is any missing field then this is returned as response:
```
        {
            "type": "missing",
            "loc": [
                "body",
                "fuelCategoryId"
            ],
            "msg": "Field required",
            "input": {
                "complianceReportId": "1",
                "id": "a9d30ac2-ba2d-47e2-a655-00e27edc71c2",
                "fuelType": "CNG",
                "fuelTypeId": 2,
                "units": "m³",
                "unrecognized": false,
                "modified": true,
                "validationStatus": "pending"
            }
        },
```
Then, frontend receives the response, and if any error, then this is catched and then we iterate over the errors returned by the backend.  We need fo have the same field in the front end. Otherwise, how are we going to map the errors?

Should we do in a different way?

Reply:
No, I was just confused because the error is working in your screenshot and the MR description doesn't mention fixing an error message so I wasn't sure why this was being changed.